jobs/pull-kubernetes-federation-e2e-gce-canary.env

```diff
@@ -0,0 +1,22 @@
+# Panic if anything mutates a shared informer cache
+ENABLE_CACHE_MUTATION_DETECTOR=true
+
+FEDERATION=true
+USE_KUBEFED=true
+PROJECT=k8s-jkns-pr-bldr-e2e-gce-fdrtn
+KUBE_REGISTRY=gcr.io/k8s-jkns-pr-bldr-e2e-gce-fdrtn
+KUBERNETES_PROVIDER=gce
+GINKGO_TEST_ARGS=--ginkgo.focus=\[Feature:Federation\] --ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[NoCluster\]
+KUBEKINS_TIMEOUT=90m
+
+# Recycle control plane.
+# We only recycle federation control plane in each run, we don't
+# want to recycle the clusters, it's too slow.
+# This is accomplished by not setting FEDERATION_CLUSTERS env
+# var or setting it to empty string. We set it to empty string
+# to explicitly call it out.
+FEDERATION_CLUSTERS=
+
+# Federation control plane options.
+DNS_ZONE_NAME=pr-bldr.test-f8n.k8s.io.
+FEDERATIONS_DOMAIN_MAP=federation=pr-bldr.test-f8n.k8s.io
```

Comment:
I don't see that in https://github.com/kubernetes/test-infra/blob/master/jobs/pull-kubernetes-federation-e2e-gce.sh?